src/learn/ExperienceModule.tsx

```diff
@@ -492,8 +519,9 @@ class ExperienceModule extends Component<ExperienceModuleProps, ExperienceModule
                 reason !== "escapeKeyDown" &&
                 this.props.user.learningPurchase === false
               ) {
+                const navigate = useNavigate();
```

Comment:
This is kinda weird -  I'm not sure if this even works? Can you confirm that this actually works, when submitting an experience?

Reply:
no, it doesnt. I didnt notice this was a class component. Ill have to change this one too, I got the order class component working. 

Now I should have it working